### PR TITLE
[DC 2019] add Red Hat Public Sector to platinum sponsors

### PR DESCRIPTION
**Pull request by robbkidd**
Added Red Hat's Public Sector group as a sponsor for DC.

Trimmed the campaign ID from their URL because this entry ought to be reusable for different events.

### CODE REVIEW

**Comment by MikeRosTX** [comment]
@robbkidd - Is there something we should be waiting on from you before merging? WIP gives us that impression, that we should be waiting on something.

**Comment by robbkidd** [comment]
@MikeRosTX Yes, this is intentionally WIP waiting for approvals from other DC organizers.

@nathenharvey @xzilla  👀❓

File: data/events/2019-washington-dc.yml [hunk 1]
```diff
@@ -310,5 +310,7 @@ sponsors:
     level: platinum
   - id: excella
     level: platinum
+  - id: redhatgov
+    level: platinum
```
Comment (bridgetkromhout) [inline data/events/2019-washington-dc.yml]: You can override the URL right here if you want to include the campaign ID.

Comment (robbkidd) [reply data/events/2019-washington-dc.yml]: SHUT THE FRONT DOOR! Nice! I'll open another PR for that. Thanks!

**Pull request merged**